packages/amplify_api/lib/src/graphql/model_mutations_factory.dart

```diff
@@ -14,7 +14,7 @@ class ModelMutationsFactory extends ModelMutationsInterface {
 
   @override
   GraphQLRequest<T> create<T extends Model>(T model) {
-    Map<String, dynamic> variables = model.toJson();
+    Map<String, dynamic> variables = {"input": model.toJson()};
```

Comment:
I have not looked at why unit tests are failing, but I would guess tests need to be updated to match this?